### PR TITLE
[VIRT for SLEM 5.4] Change package name back because autotest can not load it

### PR DESCRIPTION
**Pull request by waynechen55**
* **Package** prepare_transactional_server.pm had been renamed to transactional::prepare_transactional_server.pm in [pr#15878](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15878), which leads to test run failure, because autotest can not load it, for example, [this failure](https://openqa.suse.de/tests/10097711). So need to change the package name back to the original one prepare_transactional_server.

* **Move** prepare_transactional_server one level up, so it can still be used in kubevirt test as the base module.

* **Verification runs:**
  * [install guest on sle-micro test can load](https://openqa.nue.suse.com/tests/10097764)
  * [kubevirt test can still use it](http://10.67.129.66/tests/1785)

### CODE REVIEW

**Comment by waynechen55** [comment]
@alice-suse @Julie-CAO @guoxuguang @nanzhg Pleas have a look.

**Review by alice-suse — APPROVED** [review]
LGTM

**Comment by alice-suse** [comment]
@nanzhg  Will wait for your review before merge :)

**Comment by nanzhg** [comment]
OK, it's good for me. LGTM

> @nanzhg Will wait for your review before merge :)

**Pull request merged**
